scripts/scil_tractogram_commit.py

```diff
@@ -240,9 +241,12 @@ def _save_results(args, tmp_dir, ext, in_hdf5_file, offsets_list, sub_dir,
     # Loading the tractogram (we never did yet! Only sent the filename to
     # commit). Reminder. If input was a hdf5, we have changed
     # args.in_tractogram to our tmp_tractogram saved in tmp_dir.
-    if ext == '.trk':
+    if ext == '.trk' or args.reference is None:
```

Comment:
That should be an AND? 
If it is not a TRK the 'same' argument cannot be used, so either that OR should never work (tck and none cannot co-exist) or it should be an AND